runtime/interpreter/value.go

```diff
@@ -1586,6 +1592,25 @@ func (v *StringValue) Iterator(_ *Interpreter) ValueIterator {
 	}
 }
 
+func (v *StringValue) ForEach(
+	interpreter *Interpreter,
+	_ sema.Type,
+	function func(value Value) (resume bool),
+	_ LocationRange,
+) {
+	iterator := v.Iterator(interpreter)
+	for {
+		value := iterator.Next(interpreter)
```

Comment:
Does `StringValue` not have an iterate function like array does?

Reply:
Unfortunately no, only `ArrayValue` and `DictionaryValue` currently have iterate functions